_tuning-your-cluster/availability-and-recovery/segment-replication/index.md

```diff
@@ -28,6 +28,13 @@ Segment replication can be applied in a variety of scenarios, including:
 - When experiencing very high loads, you want to add new nodes but don't want to index all data immediately.
 - OpenSearch cluster deployments with low replica counts, such as those used for log analytics.
 
+## Remote-backed storage
+
+As of OpenSearch 2.10, you can use two methods for segment replication:
+
+- With **remote-backed storage**, a persistent storage solution: The primary shard sends segment files to the remote-backed storage and the replica shards source the copy from the same store. For more information about using a remote-backed storage, see [Remote-backed storage]({{site.url}}{{site.baseurl}}/tuning-your-cluster/availability-and-recovery/remote-store/index/#segment-replication-and-remote-backed-storage).
+- Without remote-backed storage: The primary shard sends segment files directly to the replica shards using node-to-node communication.
```

Comment:
Can we call the second method what it is, as opposed to what it isn't? Node-to-node replication?